converters/cffunction.cfc

```diff
@@ -40,7 +40,9 @@ component extends="BaseConverter" {
 			s = s & "public ";
 		}
 		if (structKeyExists(attr, "returntype")) {
-			s = s & attr.returntype & " ";
+			if( !variables.options.cleanTypes || ( attr.returntype != "VOID" && attr.returntype != "any" ) ) {
```

Comment:
Since `any` is the default, it's fine to optionally exclude, but removing `void` changes the semantics. Also, it's unnecessary to nest another `if` here.